Assets/Player.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Player : MonoBehaviour
+{
+    public int maxHealth = 100;
```

Comment:
It's better to use `[SerializeField] private` instead of `public` for variables that appear in the Editor. Then, instance variables are not automatically available outside of their class.